examples/standalone_flow_matching.ipynb

```diff
@@ -27,14 +27,19 @@
     "            nn.Linear(h, h), nn.ELU(),\n",
     "            nn.Linear(h, h), nn.ELU(),\n",
     "            nn.Linear(h, dim))\n",
-    "    \n",
+    "\n",
+    "    # Batch-aware forward call\n",
     "    def forward(self, t: Tensor, x_t: Tensor) -> Tensor:\n",
     "        return self.net(torch.cat((t, x_t), -1))\n",
-    "    \n",
-    "    def step(self, x_t: Tensor, t_start: Tensor, t_end: Tensor) -> Tensor:\n",
+    "\n",
+    "    # ODE-solve step for sampling\n",
+    "    def step(self, x_t: Tensor, t_start: Tensor, t_end: Tensor) -> Tensor:      \n",
     "        t_start = t_start.view(1, 1).expand(x_t.shape[0], 1)\n",
```

Comment:
```suggestion
    "    def step(self, x_t: Tensor, t_start: Tensor, t_end: Tensor) -> Tensor:      \n",
    "        """ODE-solve step for sampling with Midpoint solver."""\n",
    "        t_start = t_start.view(1, 1).expand(x_t.shape[0], 1)\n",
```

Let's move this comment into the function docstring and combine with the comment on line 39.